<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="R8-6R5ZK" name="com.ecofactorqa.stateapi.StateAPI_Set_Invalid_Cool_Setpoint" tests="12" failures="0" timestamp="5 May 2015 22:57:34 GMT" time="6.706" errors="3">
  <testcase name="set_invalid_coolsetpoint_null" time="0.040" classname="com.ecofactorqa.stateapi.StateAPI_Set_Invalid_Cool_Setpoint">
    <error type="java.lang.AssertionError" message="Expected status 400. Actual status is :200 expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Expected status 400. Actual status is :200 expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.ecofactorqa.stateapi.StateAPI_Set_Invalid_Cool_Setpoint.set_invalid_coolsetpoint_null(StateAPI_Set_Invalid_Cool_Setpoint.java:80)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- set_invalid_coolsetpoint_null -->
  <testcase name="set_invalid_coolsetpoint_zero" time="0.040" classname="com.ecofactorqa.stateapi.StateAPI_Set_Invalid_Cool_Setpoint"/>
  <testcase name="set_invalid_coolsetpoint_decimalcoma" time="0.020" classname="com.ecofactorqa.stateapi.StateAPI_Set_Invalid_Cool_Setpoint"/>
  <testcase name="set_valid_heatsetpoint" time="0.041" classname="com.ecofactorqa.stateapi.StateAPI_Set_Invalid_Cool_Setpoint"/>
  <testcase name="set_invalid_coolsetpoint_boundaryhigh" time="0.040" classname="com.ecofactorqa.stateapi.StateAPI_Set_Invalid_Cool_Setpoint"/>
  <testcase name="set_invalid_coolsetpoint_boundarylow" time="0.050" classname="com.ecofactorqa.stateapi.StateAPI_Set_Invalid_Cool_Setpoint"/>
  <testcase name="set_invalid_coolsetpoint_empty" time="0.030" classname="com.ecofactorqa.stateapi.StateAPI_Set_Invalid_Cool_Setpoint"/>
  <testcase name="set_invalid_coolsetpoint_doublequotes" time="1.680" classname="com.ecofactorqa.stateapi.StateAPI_Set_Invalid_Cool_Setpoint">
    <error type="java.lang.AssertionError" message="Expected status 400. Actual status is :200 expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Expected status 400. Actual status is :200 expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.ecofactorqa.stateapi.StateAPI_Set_Invalid_Cool_Setpoint.set_invalid_coolsetpoint_doublequotes(StateAPI_Set_Invalid_Cool_Setpoint.java:135)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- set_invalid_coolsetpoint_doublequotes -->
  <testcase name="set_invalid_coolsetpoint_singlequotes" time="0.040" classname="com.ecofactorqa.stateapi.StateAPI_Set_Invalid_Cool_Setpoint"/>
  <testcase name="Get_thermostat_mode" time="0.095" classname="com.ecofactorqa.stateapi.StateAPI_Set_Invalid_Cool_Setpoint"/>
  <testcase name="set_invalid_coolsetpoint_decimaldot" time="1.550" classname="com.ecofactorqa.stateapi.StateAPI_Set_Invalid_Cool_Setpoint">
    <error type="java.lang.AssertionError" message="Expected status 400. Actual status is :200 expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Expected status 400. Actual status is :200 expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.ecofactorqa.stateapi.StateAPI_Set_Invalid_Cool_Setpoint.set_invalid_coolsetpoint_decimaldot(StateAPI_Set_Invalid_Cool_Setpoint.java:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- set_invalid_coolsetpoint_decimaldot -->
  <testcase name="set_thermostat_to_cool_mode" time="3.080" classname="com.ecofactorqa.stateapi.StateAPI_Set_Invalid_Cool_Setpoint"/>
</testsuite> <!-- com.ecofactorqa.stateapi.StateAPI_Set_Invalid_Cool_Setpoint -->
